### PR TITLE
Enable postgres and sqlserver tests where able

### PR DESCRIPTION
**Pull request by gjwatts**
All the Postgres tests work now, with the exception of one (DataSourceTest:testXARecovery), which Kyle will be fixing after I get this branch checked in.

### CODE REVIEW

**Comment by gjwatts** [comment]
#build

File: dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml [hunk 1]
```diff
@@ -261,6 +261,10 @@
 	<javaPermission className="java.lang.RuntimePermission" name="accessDeclaredMembers" />
 	<!-- Postgres and SQLServer permission to make a basic query-->
 	<javaPermission className="java.util.PropertyPermission" name="*" actions="read"/>
+
+	<!-- Postgres permission during commit cycle -->
+	<javaPermission className="org.osgi.framework.ServicePermission" name="*" actions="get"/>
+	<javaPermission className="javax.security.auth.PrivateCredentialPermission"  name='* * "*"' actions="read" />
```
Comment (KyleAure) [inline dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml]: does `name="*"` not work here?

File: dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java [hunk 2]
```diff
@@ -215,20 +215,36 @@
...
+                    }
+                }
+            }
+
+            try (Statement st = con.createStatement()) {
+                if (tableFound) {
+                    st.executeUpdate("drop table cities");
+                }
+                String dbProductName = md.getDatabaseProductName().toUpperCase();
+                if (dbProductName.contains("IDS") || dbProductName.contains("INFORMIX")) // Informix JCC and JDBC
```
Comment (KyleAure) [inline dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java]: We no longer test against Informix so this logic can be cut out.

File: dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml (hunk 1 above)
Comment (gjwatts) [reply dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml]: I don't believe so.  I can run that test tomorrow. I tried various combinations and nothing worked.  I then did some looking in other places and found that line.  I copied it over and that worked.

File: dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java (hunk 2 above)
Comment (gjwatts) [reply dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java]: There are multiple places where Informix is referenced.  Should I remove them all in DataSourceTestServlet, or just the one you highlighted?

Comment (KyleAure) [reply dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java]: Yeah we can remove anything that references Informix in this test bucket.  But, that can be done in a future commit.

File: dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml (hunk 1 above)
Comment (KyleAure) [reply dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml]: If it works we can leave it.  But from my understanding only the last * in a java permission statement actually does anything.

Comment (gjwatts) [reply dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml]: I changed that javaPermission line to name="*" and re-ran the FAT test bucket and it failed with java 2 security errors.  I changed it back to what I have above and the FAT test bucket passed.

**Comment by gjwatts** [comment]
#libbybot

**Pull request merged**